gradoop-store/gradoop-hbase/src/test/java/org/gradoop/storage/impl/hbase/filter/impl/HBasePropEqualsTest.java

```diff
@@ -0,0 +1,128 @@
+package org.gradoop.storage.impl.hbase.filter.impl;
+
+import org.apache.hadoop.hbase.filter.CompareFilter;
+import org.apache.hadoop.hbase.filter.FilterList;
+import org.apache.hadoop.hbase.filter.SingleColumnValueFilter;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.gradoop.common.GradoopTestUtils;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.common.model.impl.properties.PropertyValue;
+import org.gradoop.common.model.impl.properties.PropertyValueUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.gradoop.storage.impl.hbase.constants.HBaseConstants.CF_PROPERTY_TYPE;
+import static org.gradoop.storage.impl.hbase.constants.HBaseConstants.CF_PROPERTY_VALUE;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test class for {@link HBasePropEquals}
+ */
+@RunWith(Parameterized.class)
+public class HBasePropEqualsTest {
+
+  /**
+   * Property type
+   */
+  private String propertyType;
+
+  /**
+   * Property key
+   */
+  private String propertyKey;
+
+  /**
+   * Property value
+   */
+  private PropertyValue propertyValue;
+
+  /**
+   * Constructor for parametrized test
+   *
+   * @param propertyKey property key to test
+   * @param value property value to test
+   */
+  public HBasePropEqualsTest(String propertyKey, Object value) {
+    this.propertyKey = propertyKey;
+    this.propertyValue = PropertyValue.create(value);
+    this.propertyType = this.propertyValue.getType() == null ?
+      "null" : this.propertyValue.getType().toString();
```

Comment:
When [#877] is merged "null" will be accessible via Enum. Add a todo to remind you to change it.

Reply:
TypeString has private access